### DISCUSSION TITLE
Re-install dokku without wiping machine?

### DISCUSSION THREAD

**Discussion by alancwoo** [dokku/dokku#5526] [Q&A]
I have a server provisioned by a client so unfortunately it's not so easy to wipe/start anew by just spinning up a new instance.

I installed dokku, then for some troubleshooting, uninstalled via `apt remove dokku` and would like to reinstall.

Unfortunately, running the bootstrap results in `Unable to correct problems, you have held broken packages.` and something like `apt install dokku` returns:

```
The following packages have unmet dependencies:
 dokku : Depends: docker-compose-plugin but it is not installable or
                  moby-compose but it is not installable
         Recommends: herokuish (>= 0.3.4) but it is not going to be installed
         Recommends: dokku-update but it is not going to be installed
         Recommends: dokku-event-listener but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

and then adding the dependencies results in the eventual `Package docker-compose-plugin is not available, but is referred to by another package.`

I've run all the apt purge/clean/fix commands I can find and am wondering if there is any way to repair this machine without having to request the client to wipe and re-provision?

**Comment by josegonzalez** [comment]
That `docker-compose-plugin` package is provided by the official docker repositories. Are you running an older server that might not have access to the newer repos?
